src/components/blocks/Event.tsx

```diff
@@ -28,13 +28,17 @@ const Event = ({ event, last, index }: EventProps) => {
   return (
     <div className={`text-center w-[140px] relative flex-shrink-0`}>
       <div className={`h-[2px] bg-blue-600 absolute top-1/2 translate-y-[-1px] z-0 ${timelineStyles}`} />
-      <div className="flex items-end justify-center h-[100px]">{!even && renderText(name, month + " " + year)}</div>
+      <div className="flex items-end justify-center h-[100px]">{!even && renderText(title, month + " " + year)}</div>
       <div className="flex place-content-center h-full relative z-10">
         <div className="circle-container">
-          {category ? <div className="circle-icon">{getCategoryIcon(category, "22")}</div> : <div className={index === 0 || last ? "circle-full" : "circle-empty"}></div>}
+          {/* {category ? (
```

Comment:
The commented code intentional?

Reply:
Just old implementation when events had categories, e.g. target event. I'll remove for now